documentation/modules/ROOT/pages/post-processors/reselect-columns.adoc

```diff
@@ -108,6 +109,9 @@ To enable the connector to use the `ReselectColumnsPostProcessor`, add the follo
 |6
 |Enables or disables reselection based event key field names.
 
+|7
+|Log or fail the connector in  case reselection fail or give empty record.
```

Comment:
Log a warning or throw an exception when reselection does not find the row or if a database exception is raised.